builtin/settingtypes.txt

```diff
@@ -2309,6 +2309,9 @@ pitch_move (Pitch move mode) bool false
 #    This requires the "fast" privilege on the server.
 fast_move (Fast movement) bool false
 
+#    Sneak Lock
+sneak_lock (Sneak lock) bool false
```

Comment:
I don't think this should get an entry here. This is not a setting that should be changed through the settings dialog.